cmd/system-probe/subcommands/debug/command.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package debug
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/command"
+	"github.com/DataDog/datadog-agent/comp/core"
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/comp/core/log"
+	"github.com/DataDog/datadog-agent/comp/core/sysprobeconfig"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// cliParams are the command-line arguments for this subcommand
+type cliParams struct {
+	*command.GlobalParams
+
+	// args contains the positional command-line arguments
+	args []string
+}
+
+// Commands returns a slice of subcommands for the 'agent' command.
+func Commands(globalParams *command.GlobalParams) []*cobra.Command {
+	cliParams := &cliParams{
+		GlobalParams: globalParams,
+	}
+	debugCommand := &cobra.Command{
+		Use:   "debug [module] [state]",
+		Short: "Print the runtime state of a running system-probe",
+		Long:  ``,
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliParams.args = args
+			return fxutil.OneShot(debugRuntime,
+				fx.Supply(cliParams),
+				fx.Supply(core.BundleParams{
+					ConfigParams:         config.NewAgentParamsWithoutSecrets(""),
+					SysprobeConfigParams: sysprobeconfig.NewParams(sysprobeconfig.WithSysProbeConfFilePath(globalParams.ConfFilePath)),
+					LogParams:            log.LogForOneShot("SYS-PROBE", "off", false),
+				}),
+				core.UseSysProbeConfigForLog(),
+				core.Bundle,
+			)
+		},
+	}
+
+	return []*cobra.Command{debugCommand}
+}
+
+func debugRuntime(sysprobeconfig sysprobeconfig.Component, cliParams *cliParams) error {
+	cfg := sysprobeconfig.Object()
+	client := api.GetClient(cfg.SocketAddress)
+
+	var path string
+	if len(cliParams.args) == 1 {
+		path = fmt.Sprintf("http://localhost/debug/%s", cliParams.args[0])
+	} else {
+		path = fmt.Sprintf("http://localhost/%s/debug/%s", cliParams.args[0], cliParams.args[1])
+	}
+
+	// TODO rather than allowing arbitrary query params, use cobra flags
```

Comment:
I realized this was just copied over from the previous version of this file, but if this is something we think is necessary we should make a ticket for it in our backlog & remove this from the code

Reply:
It is very minor. I'm OK leaving a code comment in context.